package.json

```diff
@@ -11,9 +11,11 @@
     "start-front": "webpack serve --config webpack-front.dev.config.js",
     "lint": "eslint \"**/*.ts\"",
     "eslint:fix": "eslint --fix \"**/*.ts\"",
-    "prettier:write": "prettier --write \"**/*.ts\"",
+    "prettier:write": "prettier --write \"**/*.(ts|js)\"",
     "test": "jest",
-    "precommit": "npm run lint && npm run test"
+    "precommit": "npm run lint && npm run test",
+    "doc:server": "./node_modules/.bin/jsdoc build/BE/index.js -d doc/BE",
```

Comment:
Better to use typedoc and run it against interfaces to generate proper docs For your example it generates empty docs